.github/workflows/e2e-tests-on-PR.yml

```diff
@@ -147,7 +264,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           working-directory: ./e2e-tests
-          command: 'npx cypress-tags run --record --parallel --ci-build-id ${{ needs.context.outputs.uuid }}'
+          record: true
+          parallel: true
+          ci-build-id: ${{ needs.context.outputs.uuid }}
```

Comment:
is that a todo ? I can't find anything on google about that, but it looks unusual 

rest looks great 👍 just curious on this line

Reply:
Treated as an import statement to import a sub-routine on line 18 of the same file.  Hit re-run on GHA does not give you an unique workflow run id, so when it tries to rerun the cypress tests as the previous one already exists it won't run them again. So we create a new unique identifier everytime GHA for the Cypress to identify it as a new run and run all the tests.